src/main/kotlin/no/nav/syfo/behandler/BehandlerService.kt

```diff
@@ -336,7 +344,7 @@ class BehandlerService(
             log.error(
                 "Persistert behandlerkontor har muligens feil herId ${existingBehandlerKontor.herId}: " +
                     "Sjekk kontor med partnerId ${existingBehandlerKontor.partnerId}." +
-                    "Adresseregisteret returnerte overordnet herId ${behandler.kontor.herId} for behandler ${behandler.behandlerRef}"
+                    "Adresseregisteret returnerte overordnet herId ${behandler.kontor.herId} for behandler herId ${behandler.herId}"
```

Comment:
For øvrig dette jeg tenkte på da jeg sa at jeg ikke liker at man gjør "ekstra" ting i PR'er. Nå er jo dette en liten endring, men det blir vanskeligere å lese git-historikken når det står at dette ble gjort for å oppdatere behandlertelefon 🙃

Reply:
Ja, litt enig. Men oppdaget bare at den uuid'en som ble logget her er fra et transient objekt og ikke er den persisterte uuid'en for behandleren.